### PR TITLE
Profile details Listing UI

### LINKED ISSUE #34: IN: Profile details page
Make profile details page dynamic based on user

### PR DESCRIPTION
**Pull request by damienaries**
### What this PR does (required):
- Route and data from listings/results page to listing detail page
- 85% complete page static UI

### Screenshots / Videos (required):
Screen Shot 2021-09-03 at 11 12 25 AM

### Any information needed to test this feature (required):
- Log in, click temporary link in nav to get to listings page, then click on a card to get to the listing details page

### Any issues with the current functionality (optional):
- Date/time pickers not functional yet, also need to implement MUI date component and link to sitter's availability

### CODE REVIEW

**Comment by damienaries** [comment]
Hey @rajivtitus! a bit last minute but here's the listing ui page

File: client/src/pages/Listings/ListingDetail/useStyles.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      height: '8rem',
+      width: '8rem',
+      margin: '.5rem 1rem .5rem 0',
+      borderRadius: theme.shape.borderRadius,
+    },
+  },
+  bookingContainer: {
+    padding: '2rem 1rem',
+    backgroundColor: '#fff',
+    // height: '60%',
```
Comment (rajivtitus) [inline client/src/pages/Listings/ListingDetail/useStyles.ts:67]: Please remove commented styles

File: client/src/App.tsx [hunk 2]
```diff
@@ -30,7 +31,7 @@ function App(): JSX.Element {
                 <Route exact path="/login" component={Login} />
                 <Route exact path="/signup" component={Signup} />
                 <Route exact path="/demo" component={Dashboard} />
-                <Route exact path="/profile" component={ProfilePhoto} />
+                {/* <Route exact path="/profile/edit-photo" component={ProfilePhoto} /> */}
```
Comment (rajivtitus) [inline client/src/App.tsx:34]: Just make sure the intended routes are working :)

**Review by rajivtitus — APPROVED** [review]
UI looks really good :)

File: client/src/App.tsx (hunk 2 above)
Comment (damienaries) [reply client/src/App.tsx:34]: Definitely. Just had it here for testing

**Comment by damienaries** [comment]
Thank you! I'll address those comments and merge asap.

**Comment by damienaries** [comment]
Merging and deleting branch

**Pull request merged**